src/helper.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  convertUrlToParams: (url) => {
+    const urlParams = url.split('?');
+    let parts = [];
+    const query = {};
+
+    if (urlParams[0] === '/') {
+      return {
+        parts,
+        query,
+      };
+    }
+
+    const queries = urlParams[1]?.split('&') || [];
```

Comment:
<img width="542" alt="Screenshot 2023-09-19 at 12 25 12" src="https://github.com/mate-academy/node_params-server/assets/53228259/18fe95b4-ce6b-4cbc-a82f-7878ae70ce0c">

you failed tests because of ?. change to
`(urlParams[1] && urlParams[1].split('&')) || [];`